docs/Architecture Decision Record/009-git-workflow.md

```diff
@@ -0,0 +1,94 @@
+5. If working locally, run `git push origin <BRANCH NAME>`
+6. Create [a draft pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests) from new branch to `raft-tdp-main`. The PR title should include WIP e.g. `WIP: Adding a feature` (If your work is dependent on another branch that has yet to be merged, issue the pull request against that branch)
+8. Implement, Test, Review work independently
+9. When finished: 
+    * Update the Pull Request Template
+    * Add in-line comment to the file changes to provide context for the proposed changes
+    * Ensure there are no merge conflicts 
+    * Ensure CI/CD pipelines are green
+    * Update the title to remove `WIP`, change the PR to [Ready for Review](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-stage-of-a-pull-request), assign label `raft-review`.
+    * If your PR is against a branch other than `raft-tdp-main` keep it as a draft PR and add the tag `blocked` until the other branch is merged. 
+13. Assign a reviewer: 
+    * For development work, assign **at least two** Raft developers with one of them being `carltonsmith` or `jtwillis92` (**but not both**).
+    * For research and design assign `shubhi-raft` and `jenewingpierce`
+    * For security controls assign `carltonsmith`
+    * For documentation submitted by OFA and 18F tag `lfrohlich` and `adpennington` and remove label `raft-review`
+    * For documentation submissions and updates by raft assign `lfrohlich` and `adpennington` and add label `QASP review`
+14. For PRs with `raft-review` label, the appropriate reviewer performs the review and/or requests changes. **SLA** 3 days: 2 days to perform the review and 1 day to implement the requested changes. 
+    * When changes are asked for, the changes are made by the contributor
+15. When satisfied, the reviewer 
+    * `approves` the PR
+    * removes `raft-review` label
+    * adds `qasp-review` label
+    * assigns 18F/HHS as the reviewer:
+        * For backend development work assign `alexsoble`, `adpennington` and `lfrohlich`
+        * For frontend work, assign the same as backend work but also include `iamjolly` for a11y review
+        * For research and design work assign `lfrohlich`
+        * For security controls assign `hursey013` and `lforhlich`
+18. For PRs with `qasp-review` label, the appropriate reviewer performs the review and/or requests changes. **SLA** 5 days: 3 days to perform the review and 2 days to implement the requested changes.
+    * When changes are asked for, the changes are made by the contributor. Raft reviewers should internalize the changes asked by 18F/HHS such that the same feedback is already incorporated and/or caught in future (continuous improvement)
+    * When security controls are approved, additional reviews are asked for by OCIO
+19. When satisfied, the 18F/HHS/OCIO reviewer `approves` the PR
```

Comment:
can we clarify here and throughout that PRs with the qasp-review label will be approved according to the qasp-checklist?

Reply:
No problem. Would probably be easiest to make a **Notes** section at the bottom and add it there, rather than adding more verbiage to each line. Thoughts?

Reply:
I added the Notes section with the appropriate update. Please let me know if there is another way you would prefer for us to address this.